crud/count.lua

```diff
@@ -0,0 +1,212 @@
+local checks = require('checks')
+local errors = require('errors')
+local vshard = require('vshard')
+local fiber = require('fiber')
+
+local call = require('crud.common.call')
+local utils = require('crud.common.utils')
+local sharding = require('crud.common.sharding')
+local filters = require('crud.compare.filters')
+local count_plan = require('crud.compare.plan')
+local dev_checks = require('crud.common.dev_checks')
+local sharding_key_module = require('crud.common.sharding_key')
+
+local compare_conditions = require('crud.compare.conditions')
+
+local CountError = errors.new_class('CountError', {capture_stack = false})
+
+local COUNT_FUNC_NAME = '_crud.count_on_storage'
+local DEFAULT_COUNT_TO_YIELD = 100
```

Comment:
Maybe 1000 is better default. 100 looks too often.

Reply:
Updated.